pkg/apiserver/registry/networkpolicy/clustergroupmember/rest.go

```diff
@@ -63,11 +63,14 @@ func (r *REST) Get(ctx context.Context, name string, options *metav1.GetOptions)
 		}
 		memberList.EffectiveIPBlocks = effectiveIPBlocks
 	} else {
-		effectiveMembers := make([]controlplane.GroupMember, 0, len(groupMembers))
+		// ClusterGroup or its childGroups are defined by selectors.
+		// An empty effectiveMembers slice signals no entities are selected.
```

Comment:
No there's no such contract. But before the change if the ClusterGroup is defined by selectors the resulting effectiveMembers will be shown as an empty slice on the client side. I just want to keep that behavior in case any external client consuming this is expecting this field to be there

Reply:
I tried an empty clusterGroup and the response I got was:
```
apiVersion: controlplane.antrea.io/v1beta2
effectiveIPBlocks: null
effectiveMembers: []
kind: ClusterGroupMembers
metadata:
  creationTimestamp: null
  name: cg
```
Since the field doesn't set `omitEmpty`, the fields will be there anyway, just the value will be null or an empty slice. I feel if they doesn't have different meaning, maybe better to unify them to avoid confusion? And the code may be neater in that way:
```
if len(ipBlocks) > 0 {
	effectiveIPBlocks := make([]controlplane.IPNet, 0, len(ipBlocks))
        ...
}
if len(groupMembers) > 0 {
	effectiveMembers := make([]controlplane.GroupMember, 0, len(groupMembers))
        ...
}
```

Reply:
Sure, changed